arctic/store/version_store.py

```diff
@@ -352,13 +352,11 @@ def _show_info(self, symbol, as_of=None):
         """
         print self._get_info(symbol, as_of)
 
-    def _get_info(self, symbol, as_of=None):
-        _version = self._read_metadata(symbol, as_of=as_of)
-        handler = self._read_handler(_version, symbol)
-        if hasattr(handler, "get_info"):
-            return handler.get_info(self._arctic_lib, _version, symbol)
-        else:
-            return """Handler: %s\n\nVersion document:\n%s""" % (handler.__class__.__name__, pprint.pformat(_version))
+    def _get_info(self, symbol, as_of=None, version=None):
```

Comment:
Elsewhere in the API we use `as_of` as the selector we use when slicing out a particular version:
   https://github.com/manahl/arctic/blob/master/arctic/store/version_store.py#L186
Is there a reason to make this take `version` too?